test/test_torch.py

```diff
@@ -7600,6 +7600,14 @@ def test_masked_scatter(self):
         with self.assertRaises(RuntimeError):
             dest.masked_scatter_(mask, src)
 
+        # test bool tensor case
+        for device in torch.testing.get_all_device_types():
```

Comment:
I don't really understand the test -- I would assume you'd just run the same tests through bool and byte and check you get the same answers?